docs/dataset/quickstart.md

```diff
@@ -5,3 +5,185 @@ icon: kolena/flame-16
 # :kolena-flame-20: Quickstart
 
 Install Kolena to set up rigorous and repeatable model testing in minutes.
+
+In this quickstart guide, we'll use we will use the [300 Faces In the Wild (300-W)](https://ibug.doc.ic.ac.uk/resources/300-W/)
+example dataset to demonstrate the how to curate test data and test models in Kolena. The examples
+use data located in a public S3 bucket `s3://kolena-public-examples`
+
+## Installing `kolena` Python SDK
+
+=== "`pip`"
+
+    ```shell
+    pip install kolena
+    ```
+
+=== "`poetry`"
+
+    ```shell
+    poetry add kolena
+    ```
+
+## Clone the examples
+
+The [kolenaIO/kolena](https://github.com/kolenaIO/kolena) repository contains a number of example integrations to clone
+and run directly:
+
+<div class="grid cards" markdown>
+
+- [:kolena-keypoint-detection-20: Example: Keypoint Detection ↗](https://github.com/kolenaIO/kolena/tree/trunk/examples/dataset/keypoint_detection)
+
+    ![Example image and five-point facial keypoints array from the 300 Faces in the Wild dataset.](../assets/images/300-W.jpg)
+
+    ---
+
+    End-to-end face recognition 1:1 using the [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/) dataset.
+- [:kolena-chat-20: Example: Question Answering ↗](https://github.com/kolenaIO/kolena/tree/trunk/examples/dataset/question_answering)
+
+    ![Example questions from CoQA dataset.](../assets/images/CoQA.jpg)
+
+    ---
+
+    Question Answering using the
+    [Conversational Question Answering (CoQA)](https://stanfordnlp.github.io/coqa/) dataset
+</div>
+
+Now, let's set up the keypoint detection example:
+
+```shell
+cd kolena/examples/object_detection_2d
+poetry update && poetry install
+```
+
+## Dataset Upload
```

Comment:
We should make these steps explicit by numbering them:

1. Upload a Dataset
2. Upload Model Results
3. Investigate Model Performance
4. Create Quality Standards
5. Compare Models

These should all be action-oriented as well — see the section headers for the existing quickstart (which also should be numbered 🤔):

<img width="204" alt="Screenshot 2024-01-04 at 2 23 19 PM" src="https://github.com/kolenaIO/kolena/assets/9648886/36e7f0d8-b73b-48e8-bf59-8bc5e821199c">